jina/peapods/runtimes/asyncio/rest/models.py

```diff
@@ -54,15 +54,60 @@ class CamelCaseConfig(BaseConfig):
     allow_population_by_field_name = True
 
 
+def _get_oneof_validator(oneof_fields: List, oneof_key: str) -> Callable:
```

Comment:
instead of using this, u can use `bind` to bind parameters to method.

`from functools import partial`

Reply:
We also need to change the qualname for each call. We can leave this for now, as it is easier to read